### ISSUE TITLE
Tile map builder

### ISSUE THREAD

**Issue by sethtrain** [sethtrain/marvin#12]
If we can do it all from one application that might be cool.

**Comment by omnomicon** [comment]
i wonder if there's room here to go more abstract? What i mean is the main complaint I have with roll20 and fantasy grounds is, you log in and are immediately confronted with hundreds of knobs and switches. I think there's room for a really simple map builder. maybe one where the tiles are generated in abstract ways. I'm thinking abstract as in Tsuro-like tiles:

[![Tsuro tiles](http://www.spelexperten.com/shop/165/art65/h4149/13084149-origpic-a6027b.jpg)]()

**Comment by sethtrain** [comment]
Yeah, I think this would be a cool idea.  I've used http://davesmapper.com/, with the kids, in the past.  It is basically the same idea.  What I'd like to think about is how to make it a bit customizable so that the DM/GM isn't hindering the narrative because the generation isn't coming out as they'd expect or want.
